### PR TITLE
Feature/sof 5049 - 2D Bandgap Notebooks

### PR DESCRIPTION
**Pull request by AcylSilane**

### CODE REVIEW

File: DigitalEcosystem/publication/bandgap2DMaterials/README.md [hunk 1]
```diff
@@ -0,0 +1 @@
+This notebook trains an XGBoost, TPOT, Roost, and SISSO model for the prediction of 2D Material bandgaps.
```
Comment (timurbazhirov) [inline DigitalEcosystem/publication/bandgap2DMaterials/README.md]: Remove

File: DigitalEcosystem/publication/bandgap2DMaterials/2dm_bandgap.py [hunk 2]
```diff
@@ -0,0 +1,588 @@
...
+
+
+# At this point, a SISSO model was run.
+
+# In[]:
+
+
+sisso_models = {
+    'r1_1term': lambda df: 5.908677334854443e+00 + \
+                           -4.182238907095930e-01 * (np.cbrt(df['ave:boiling_point'])),
```
Comment (timurbazhirov) [inline DigitalEcosystem/publication/bandgap2DMaterials/2dm_bandgap.py:501]: How did we get these "magical" numbers from the SISSO log? Please explain which log these are from in a comment.

File: DigitalEcosystem/publication/bandgap2DMaterials/tpot_autogenerated_pipeline.py [hunk 3]
```diff
@@ -0,0 +1,24 @@
+import numpy as np
```
Comment (timurbazhirov) [inline DigitalEcosystem/publication/bandgap2DMaterials/tpot_autogenerated_pipeline.py:1]: (1) https://github.com/Exabyte-io/Scientific-Projects/pull/28/files#r724591915
(2) Duplicate code again - can we reuse some of this???

File: DigitalEcosystem/publication/bandgap2DMaterials/README.md (hunk 1 above)
Comment (AcylSilane) [reply DigitalEcosystem/publication/bandgap2DMaterials/README.md]: It is gone

File: DigitalEcosystem/publication/bandgap2DMaterials/tpot_autogenerated_pipeline.py (hunk 3 above)
Comment (AcylSilane) [reply DigitalEcosystem/publication/bandgap2DMaterials/tpot_autogenerated_pipeline.py:1]: None of the code in this file can be re-used or modified by us, it is all automatically generated output from TPOT.

File: DigitalEcosystem/publication/bandgap2DMaterials/2dm_bandgap.py (hunk 2 above)
Comment (AcylSilane) [reply DigitalEcosystem/publication/bandgap2DMaterials/2dm_bandgap.py:501]: Added a note above the cell explaining where they're from

File: DigitalEcosystem/publication/bandgap2DMaterials/tpot_autogenerated_pipeline.py (hunk 3 above)
Comment (timurbazhirov) [reply DigitalEcosystem/publication/bandgap2DMaterials/tpot_autogenerated_pipeline.py:1]: https://github.com/Exabyte-io/Scientific-Projects/pull/28/files#r724616133

**Comment by AcylSilane** [comment]
Merging per https://exabyte.atlassian.net/browse/SOF-5049?focusedCommentId=42538

**Pull request merged**